listings/ch00-introduction/if_else/src/access_control.cairo

```diff
@@ -0,0 +1,23 @@
+use starknet::ContractAddress;
+
+#[contract]
+mod SimpleAccessControl {
+    use starknet::get_caller_address;
+
+    struct Storage {
+        owner: starknet::ContractAddress, 
+    }
+
+    #[constructor]
+    fn constructor(_address: starknet::ContractAddress) {
+        owner::write(_address);
+    }
+
+    #[external]
```

Comment:
this should be a `#[view]` function, given that there is no associated state change